### PR TITLE
[Rust] Updated API docs and the Rust section of WasmEdge Book 

### PR DESCRIPTION
**Pull request by apepkuss**
In this PR, the API docs of Rust bindings and the rust section of WasmEdge Book are updated.

### CODE REVIEW

**Comment by apepkuss** [comment]
Hi @hydai, could you please help review this PR? Thanks!

File: bindings/rust/wasmedge-sys/src/lib.rs [hunk 1]
```diff
@@ -14,16 +14,50 @@
...
+//! ## Build
 //!
-//! To use or build the `wasmedge-sys` crate, the `WasmEdge` library is required. The [*Build wasmedge-sys crate*](https://wasmedge.org/book/en/embed/rust.html#build-wasmedge-sys-crate) section of [WasmEdge Docs](https://wasmedge.org/book/en/) gives the tips.
+//! To use or build the `wasmedge-sys` crate, the `WasmEdge` library is required.
 //!
-//! ## A quick-start example
+//!  - If you choose to use [install.sh](https://github.com/WasmEdge/WasmEdge/blob/master/utils/install.sh) to install WasmEdge Runtime on your local system. Please use `WASMEDGE_INCLUDE_DIR` and `WASMEDGE_LIB_DIR` to specify the paths to the `include` and `lib` directories, respectively. For example, use the following commands to specify the paths after using `bash install.sh --path=/root/wasmedge-install` to install WasmEdge Runtime on Ubuntu 20.04:
+//!
+//!    ```bash
+//!    export WASMEDGE_INCLUDE_DIR=/root/wasmedge-install/include
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/lib.rs:29]: I don't think we should use `root` as an example. Maybe changing to `$HOME` is better?

File: bindings/rust/wasmedge-sys/README.md [hunk 2]
```diff
@@ -4,15 +4,54 @@ The [wasmedge-sys](https://crates.io/crates/wasmedge-sys) crate defines a group
...
+| 0.1.0         | 0.10.0        | 0.7           | 0.1           |
+
+## Build
+
+To use or build the `wasmedge-sys` crate, the `WasmEdge` library is required.
+
+- If you choose to use [install.sh](https://github.com/WasmEdge/WasmEdge/blob/master/utils/install.sh) to install WasmEdge Runtime on your local system. Please use `WASMEDGE_INCLUDE_DIR` and `WASMEDGE_LIB_DIR` to specify the paths to the `include` and `lib` directories, respectively. For example, use the following commands to specify the paths after using `bash install.sh --path=/root/wasmedge-install` to install WasmEdge Runtime on Ubuntu 20.04:
+
+    ```bash
+    export WASMEDGE_INCLUDE_DIR=/root/wasmedge-install/include
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/README.md:25]: Ditto.

File: bindings/rust/wasmedge-sdk/src/lib.rs [hunk 3]
```diff
@@ -16,42 +16,50 @@
...
+//! | 0.1.0         | 0.10.0        | 0.7           | 0.1           |
+//!
+//! ## Build
+//!
+//! To use or build the `wasmedge-sdk` crate, the `WasmEdge` library is required.
+//!
+//!  - If you choose to use [install.sh](https://github.com/WasmEdge/WasmEdge/blob/master/utils/install.sh) to install WasmEdge Runtime on your local system. Please use `WASMEDGE_INCLUDE_DIR` and `WASMEDGE_LIB_DIR` to specify the paths to the `include` and `lib` directories, respectively. For example, use the following commands to specify the paths after using `bash install.sh --path=/root/wasmedge-install` to install WasmEdge Runtime on Ubuntu 20.04:
+//!
+//!    ```bash
+//!    export WASMEDGE_INCLUDE_DIR=/root/wasmedge-install/include
```
Comment (hydai) [inline bindings/rust/wasmedge-sdk/src/lib.rs:31]: Ditto.

File: docs/book/en/src/embed/rust.md [hunk 4]
```diff
@@ -101,44 +102,12 @@ To use `wasmedge-sdk` in your project, you should finish the followign two steps
...
-
-    Then set `WASMEDGE_INCLUDE_DIR` and `WASMEDGE_LIB_DIR` environment variables to specify the `include` and `lib` (here is `lib64`) directories.
-
-      ```bash
-      root@0a877562f39e:~/workspace/me/WasmEdge/bindings/rust/wasmedge-sys# export WASMEDGE_INCLUDE_DIR=/root/workspace/me/WasmEdge-0.10.0-Linux/include/wasmedge
-      
-      root@0a877562f39e:~/workspace/me/WasmEdge/bindings/rust/wasmedge-sys# export WASMEDGE_LIB_DIR=/root/workspace/me/WasmEdge-0.10.0-Linux/lib64
-      ```
+    ```bash
+    export WASMEDGE_INCLUDE_DIR=/root/wasmedge-install/include
```
Comment (hydai) [inline docs/book/en/src/embed/rust.md:108]: Ditto.

**Review by hydai — CHANGES_REQUESTED** [review]

File: bindings/rust/wasmedge-sys/src/lib.rs (hunk 1 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sys/src/lib.rs:29]: Accepted, thanks!

File: bindings/rust/wasmedge-sys/README.md (hunk 2 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sys/README.md:25]: Accepted, thanks!

File: bindings/rust/wasmedge-sdk/src/lib.rs (hunk 3 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/src/lib.rs:31]: Accepted, thanks!

File: docs/book/en/src/embed/rust.md (hunk 4 above)
Comment (apepkuss) [reply docs/book/en/src/embed/rust.md:108]: Accepted, thanks!

**Comment by apepkuss** [comment]
@hydai The review issues have been fixed. Thanks a lot!

**Review by hydai — APPROVED** [review]

**Comment by apepkuss** [comment]
@hydai Thanks for the review!

**Pull request merged**